pip-287/src/SurveyComponent.js

```diff
@@ -0,0 +1,375 @@
+import React from 'react';
+import { Component } from 'react';
+import QuestionCard from './QuestionCard.js';
+import { Droppable, Draggable, DragDropContext } from 'react-beautiful-dnd';
+import { MDBBtn, MDBContainer, MDBInput, MDBPagination, MDBPageItem, MDBPageNav, MDBCol, MDBRow } from "mdbreact";
+import { toast } from 'react-toastify';
+import { Redirect } from "react-router-dom";
+
+import 'react-toastify/dist/ReactToastify.css';
+import '@fortawesome/fontawesome-free/css/all.min.css';
+import 'bootstrap-css-only/css/bootstrap.min.css';
+import 'mdbreact/dist/css/mdb.css';
+import './css/style.css';
+
+toast.configure();
```

Comment:
Czemu tutaj a nie np w konstruktorze ? :thinking:

Reply:
Ponieważ w dokumentacji toastyfi, zalecane jest aby konfigurować toasty na początku aplikacji

Reply:
Podany jest powód ?
Pokażesz mi to ?

Reply:
https://github.com/fkhadra/react-toastify#what-if-i-told-you-that-the-toastcontainer-is-optional

`// Call it once in your app. At the root of your app is the best place`
` toast.configure()`